keyboards/converter/next_usb/matrix.c

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Could you convert this to use matrix lite? 

https://docs.qmk.fm/#/custom_matrix?id=lite